pkg/corerp/frontend/controller/secretstores/kubernetes.go

```diff
@@ -194,6 +198,14 @@ func UpsertSecret(ctx context.Context, newResource, old *datamodel.SecretStore,
 			return nil, err
 		}
 	}
+	err = options.KubeClient.Create(ctx, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: ns}})
```

Comment:
What's the motivation to directly use create instead of Get API to check if the namespace already exists or not?

Reply:
I see we are using this patter for kubernetes objects in other areas of our codebase, so I'm guessing that's maybe why

Reply:
thats correct, we use this format in our code and couldnt find ways to only get the namespace using runtimeClinet.